### PR TITLE
bit-functions-and-operators: fix PDF build issue

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Fix https://do.pingcap.net/jenkins/blue/organizations/jenkins/pingcap%2Fdocs%2Fmerged_update_docs/detail/merged_update_docs/1335/pipeline/

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/18442
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lilin90 — APPROVED** [review]
/lgtm

File: functions-and-operators/bit-functions-and-operators.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ TiDB 支持使用 MySQL 8.0 中提供的所有[位函数和操作符](https://de
 | 函数和操作符名 | 功能描述 |
 | -------------- | ------------------------------------- |
 | [`BIT_COUNT()`](#bit_count) | 返回参数二进制表示中为 1 的个数 |
-| [`&`](#按位与) | 按位与 |
-| [`~`](#按位取反) | 按位取反 |
-| [`\|`](#按位或) | 按位或 |
+| [<code>&</code>](#按位与) | 按位与 |
+| [<code>~</code>](#按位取反) | 按位取反 |
+| [<code>\|<code>](#按位或) | 按位或 |
```
Comment (qiancai) [inline functions-and-operators/bit-functions-and-operators.md:18]:
```suggestion
| [<code>\|</code>](#按位或) | 按位或 |
```

**Pull request merged**